apcd-cms/src/apps/utils/apcd_database.py

```diff
@@ -749,15 +747,15 @@ def create_threshold_exception(form, sub_data):
             sub_data[2],
             sub_data[3],
             _clean_value(form['requestor-name']),
-            _clean_email(form['requestor-email']),
+            form['requestor-email'],
```

Comment:
Same as what I referred to in my previous comment about form validation.

Reply:
So the clean value function when texted will remove everything before the @ sign. I think this needs its own custom util

Reply:
Put a new function to clean email and tested on DB. Looks like it works and posts.